docs/en/operation/Basic-Logging.md

```diff
@@ -0,0 +1,208 @@
+---
+layout: global
+title: Basic Logging
+nickname: Basic Logging
+group: Operations
+priority: 11
+---
+
+* Table of Contents
+{:toc}
+
+## Introduction
+
+This page describes the basic logging function provided by Alluxio server processes (e.g., masters, workers and etc)
+and application processes utilizing Alluxio clients (e.g., Spark or MapReduce jobs running on Alluxio).
+Note that, there is an advanced feature that streams server-side and client-side logs to separate Alluxio logservers
+(see [remote logging]({{ '/en/operation/Remote-Logging.html' | relativize_url }}) for more details).
+
+Alluxio logging is implemented using [`log4j`](https://logging.apache.org/log4j/) and thus
+most of the configuration is done through `log4j.properties`.
+
+## Log Location
+
+### Server Logs
+
+Log files for each individual Alluxio server process (e.g., master, worker, FUSE, proxy) can be found under `${ALLUXIO_HOME}/logs/`,
+with file names like `master.log`, `worker.log`, `fuse.log` and `proxy.log`.
+
+This location can be customized by setting environment variable `ALLUXIO_LOGS_DIR`
+or appending `alluxio.logs.dir` property through JVM properties (e.g. `ALLUXIO_JAVA_OPTS+="-Dalluxio.logs.dir=/foo"`)
+See the
+[configuration settings page]({{ '/en/operation/Configuration.html' | relativize_url }}#environment-variables)
+for more information.
+
+### Application Logs
+
+Log files for the Alluxio client utilized by different applications are located with their respective application logs.
+Check out particular compute frameworks with information
+on where logs may be found.
+
+Here are documentation to configure individual application logs including
+[Apache Hadoop]({{ '/en/compute/Hadoop-MapReduce.html' | relativize_url }}#logging-configuration),
+[Apache HBase]({{ '/en/compute/HBase.html' | relativize_url }}#logging-configuration),
+[Apache Hive]({{ '/en/compute/Hive.html' | relativize_url }}#logging-configuration),
+[Apache Spark]({{ '/en/compute/Spark.html' | relativize_url }}#logging-configuration).
+
+## Configuring Log Levels
+
+Alluxio uses the following four logging levels:
+- `DEBUG`: fine-grained information, most useful for debugging purpose.
+- `INFO`: messages that highlight the status of progress.
+- `WARN`: potentially harmful events that users may need to know but the process will still continue running.
+- `ERROR`: system errors that users should pay attention to.
+
+By default, Alluxio server processes write logs at level `INFO`, which includes all events at level `INFO`, `WARN` and `ERROR`.
+
+
+### Modifying Logging with `log4j.properties`
+
+You can modify `${ALLUXIO_HOME}/conf/log4j.properties` to customize logging levels.
+
+For example, to modify the level for all logs to `DEBUG`, change the
+`rootLogger` level by modifying the first line of `log4j.properties`:
+
+```properties
+log4j.rootLogger=DEBUG, ${alluxio.logger.type}, ${alluxio.remote.logger.type}`
```

Comment:
extra \` character at the end of the line

Reply:
removed